css/style.css

```diff
@@ -0,0 +1,508 @@
+.container {
+  background: url(../img/Background.png) no-repeat;
+  background-size: cover;
+  display: flex;
+  max-width: 1440px;
+  min-width: 320px;
+  min-height: 1062px;
+  margin: auto;
+}
+
+.menu { 
+  margin: 130px 100px 0;
+}
+
+.title {
+  color: #FFF;
```

Comment:
Codacy found an issue: [Expected "#FFF" to be "#fff" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/Olga_FinalGame_JS/pullRequest?prid=5821092)